src/options.rs

```diff
@@ -14,27 +14,31 @@ impl Default for SetterPattern {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 pub struct Options {
     /// e.g. `#[setters]` (defaults to true)
     setter_enabled: bool,
     /// e.g. `#[setters(owned)]` (defaults to mutable)
     setter_pattern: SetterPattern,
     /// e.g. `#[setters(prefix="with")]` (defaults to None)
-    setter_prefix: Option<String>,
+    setter_prefix: String,
     /// e.g. `#[setters(private)]` (defaults to public)
     setter_public: bool,
     /// e.g. `#[setters(options="implicit")]` (defaults to explicit)
     setter_implicit_options: bool,
     /// e.g. `#[getters]` (defaults to false)
     getter_enabled: bool,
     /// e.g. `#[getters(prefix="with")]` (defaults to None)
```

Comment:
+1 for explaining it in the documentation. Will do that.

This place here was not meant as a user documentation, but to help other contributors put the pieces together, if they are digging in our code. Should I switch to comments instead of doc-comments here?

Do you like that I automatically added the underscore as a separator, or would you prefer that users need to provide the full prefix including the underscore (more flexible, but more verbose)?